pkg/api/scylla/v1/cluster_types.go

```diff
@@ -80,6 +80,11 @@ type ScyllaClusterSpec struct {
 	// When Scylla Manager is not installed, these will be ignored.
 	// +optional
 	Backups []BackupTaskSpec `json:"backups,omitempty"`
+	// ImagePullSecrets for Scylla and Agent images.
+	// +patchMergeKey=name
+	// +patchStrategy=merge
+	// +optional
+	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
```

Comment:
Strategic merge patch is not supported on CRDs so patchStrategy and patchMergeKey tags aren't controlling anything. Let's keep tags clean.

Reply:
done